kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -294,20 +296,28 @@ object KyuubiConf {
       .version("1.4.0")
       .fallbackConf(FRONTEND_BIND_HOST)
 
-  @deprecated(s"using ${FRONTEND_THRIFT_BINARY_BIND_PORT.key} instead", "1.4.0")
-  val FRONTEND_BIND_PORT: ConfigEntry[Int] = buildConf("frontend.bind.port")
-    .doc("(deprecated) Port of the machine on which to run the thrift frontend service " +
-      "via binary protocol.")
-    .version("1.0.0")
-    .intConf
-    .checkValue(p => p == 0 || (p > 1024 && p < 65535), "Invalid Port number")
-    .createWithDefault(10009)
-
   val FRONTEND_THRIFT_BINARY_BIND_PORT: ConfigEntry[Int] =
     buildConf("frontend.thrift.binary.bind.port")
       .doc("Port of the machine on which to run the thrift frontend service via binary protocol.")
       .version("1.4.0")
-      .fallbackConf(FRONTEND_BIND_PORT)
+      .intConf
+      .checkValue(p => p == 0 || (p > 1024 && p < 65535), "Invalid Port number")
+      .createWithDefault(10009)
+
+  val FRONTEND_THRIFT_HTTP_BIND_HOST: ConfigEntry[Option[String]] =
+    buildConf("frontend.thrift.http.bind.host")
```

Comment:
Literally, we don't need a host config for every type of FrontendServices, different ports actually meet to goal alreadly

Reply:
@yaooqinn, I found every type of FrontendServices has the host config. Does every type of FrontendServices need to use the unfied host config?

Reply:
not have a strong option on this, let's keep it